PredixMobileUIKit/PredixBarChartView.swift

```diff
@@ -0,0 +1,322 @@
+//
+//  PredixBarChartView.swift
+//  PredixMobileUIKit
+//
+//  Created by Fouche, George (GE Healthcare) on 11/15/17.
+//  Copyright © 2017 GE. All rights reserved.
+//
+
+import Charts
+import Foundation
+import PredixSDK
+
+// MARK: - The Option Enum
+
+/// List of options that can be used on the Bar Chart.
+public enum Option {
+    /// toggle on or off, Bars value on the chart.
+    case toggleValues
+    /// toggle on or off, Bar borders on the chart.
+    case toggleBarBorders
+    /// enable the legend display on the chart.
+    case enableLegend
+    /// disable the legend display on the chart.
+    case disableLegend
+    /// disable side labels on the chart.
+    case disableSideLabels
+    /// enable side labels on the chart.
+    case enableSideLabels
+    /// remove Limit Line on the chart.
+    case removeLimitLine
+    /// animate the x axis on the chart.
+    case animateX
+    /// animate the y axis on the chart.
+    case animateY
+    /// animate both the x and y axies on the chart.
+    case animateXY
+}
+
+// MARK: - The Bar Class
+
+/// To create a Bar Chart from the PredixBarChartView class, the Bar class is used to populate each data Bar on the chart.
+public class Bar {
+    var values: [Double]
+    var label: String
+    /// If colors is not provided the default color is used.
+    var colors: [UIColor] = []
+
+    /// Initialize the values, label and color of the bar.
+    /// - parameter values: the values the bar hold.
+    /// - parameter label: the bar label.
+    /// - parameter color: the bar color based on the class UIColor.
+    public init(_ values: [Double], label: String, colors: [UIColor]) {
+        self.values = values
+        self.label = label
+        self.colors = colors
+    }
+
+    /// Initialize the values and label of the bar.
+    /// - parameter Values: the values the bar hold.
+    /// - parameter label: the bar label.
+    public init(_ values: [Double], label: String) {
```

Comment:
There's no unit tests covering this initializer. What happens if the colors array is empty?